website/blog/2024-08-12-release-0.75.md

```diff
@@ -0,0 +1,490 @@
+---
+title: 'React Native 0.75 - Support for % Values in Layout, New Architecture Stabilization, Template & init Updates, and more'
+authors:
+  [
+    gabrieldonadel,
+    siddharthkul,
+    titozzz,
+    blakef,
+    cipolleschi,
+    cortinico,
+  ]
+tags: [engineering]
+date: 2024-08-14
+---
+
+Today we are excited to release React Native 0.75!
+
+This release ships several features, such as Yoga 3.1 with support for `%` values, several stabilization fixes for the New Architecture, and the introduction of the recommendation for users to use a React Native Framework.
+
+### Highlights
+
+- [Yoga 3.1 and Layout Improvements](/blog/2024/08/12/release-0.75#yoga-31-and-layout-improvements)
+- [New Architecture Stabilization](/blog/2024/08/12/release-0.75#new-architecture-stabilization)
+- [Using Frameworks](/blog/2024/08/12/release-0.75#using-frameworks)
+
+### Breaking Changes
+
+- [Touchables in TypeScript can’t be used as types in Generic expressions anymore](/blog/2024/08/12/release-0.75#touchables-in-typescript-cant-be-used-as-types-in-generic-expressions-anymore)
+- [Last version supporting minSdk 23 and minIOSVersion 13.4](/blog/2024/08/12/release-0.75#last-version-supporting-minsdk-23-and-miniosversion-134)
+- [Android: JSIModule has been deleted](/blog/2024/08/12/release-0.75#android-jsimodule-has-been-deleted)
+- [Android: PopUp Menu removed from core](/blog/2024/08/12/release-0.75#android-popup-menu-moved-to-separate-package)
+- [iOS: Finalized Push Notifications deprecation work](/blog/2024/08/12/release-0.75#ios-finalized-pushnotificationios-deprecation-work)
+- [Community CLI: Removal of ram-bundle and profile-hermes commands](/blog/2024/08/12/release-0.75#community-cli-removal-of-ram-bundle-and-profile-hermes-commands)
+
+<!--truncate-->
+
+## Highlights
+
+### Yoga 3.1 and Layout Improvements
+
+Since we last shipped Yoga [version 3.0](https://reactnative.dev/blog/2024/04/22/release-0.74#yoga-30) in React Native 0.74, we kept on pushing many improvements and new layout capabilities for your applications. React Native 0.75 ships with Yoga 3.1 and you can learn more about what’s new in the official Yoga’s [release blog post](https://www.yogalayout.dev/blog/announcing-yoga-3.1).
+
+One notable and highly requested feature is the support for `%` values in various places, such as `gaps` and `translation`
+
+:::info
+These features are available only for the New Architecture. If you are keen to use them, please consider migrating to it.
+:::
+
+#### `%` Values in Gaps
```

Comment:
```suggestion
#### Percentage Values in Gaps
```

Percent character looks a bit off at the beginning of headers, maybe we should use the word-based alternative?